restructuredtext_lint/lint.py

```diff
@@ -80,9 +75,24 @@ def error_collector(data):
     return errors
 
 
-def lint_file(filepath, encoding=None):
+def lint(content, filepath=None, ignore_sphinx=True):
+    """Lint reStructuredText and return errors
+
+    :param string content: reStructuredText to be linted
+    :param string filepath: Optional path to file, this will be returned as the source
+    :rtype list: List of errors. Each error will contain a line, source (filepath),
+        message (error message), and full message (error message + source lines)
+    """
+    if ignore_sphinx:
+        with sphinx.register_unregister_ignores():
+            return _lint(content, filepath=filepath)
+    else:
+        return _lint(content, filepath=filepath)
```

Comment:
Instead of passing through `filepath=filepath`, we should pass it as a kwarg to `_lint` since the lower level knows what the fallback should be and we don't need to double define it.

```python
def lint(content, ignore_sphinx=True, **kwargs):
  if ignore_sphinx:
    return _lint(content, **kwargs)
```